uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/zones/OrchestratorZoneControllerMockMvcTests.java

```diff
@@ -258,7 +258,9 @@ void testDeleteZone_ZoneNotFound() throws Exception {
     void testDeleteZone_Forbidden() throws Exception {
```

Comment:
Can we have a test case creating a client with mandatory authorities along with 'orchestrator.zones.write'? Since our main authority is the 'orchestrator.zones.write' and also we are going to share this to orchestrator team.